### PR TITLE
Translate fukuoka-ruby-award 2019 (ko)

### PR DESCRIPTION
**Pull request by riseshia**
Ref: https://github.com/ruby/www.ruby-lang.org/issues/1894

Translation of https://github.com/ruby/www.ruby-lang.org/pull/1897

Check a87ef3c5 for review, @ruby/www-ruby-lang-org-i18n-ko 🙏

### CODE REVIEW

File: ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+layout: news_post
+title: "2018 후쿠오카 루비 경진대회 - Matz가 심사합니다"
+author: "Fukuoka Ruby"
+translator: "shia"
+date: 2018-11-29 00:00:00 +0000
+lang: ko
+---
+
+루비스트 여러분에게,
```
Comment (marocchino) [inline ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]: 뒤의 문장과 연결 되지 않으므로 '~께'가 자연스러울 것 같습니다.

```suggestion
 루비스트 여러분께,
```

File: ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Money Forward 상의 수상자는 다음을 받게 됩니다.
+
+* Money Forward에서 일하고 있는 루비 커미터와의 저녁 식사
+* 개인 재정 관리 서비스 'Money Forward'의 10년 프리미엄 무료 이용권
+
+Salesforce 상의 수상자는 다음을 받게 됩니다.
+
+* salesforce.com의 기념품
+
+'Matz는 코드를 주의깊게 테스트하고 읽어볼 것이므로, 지원할만한 충분한 가치가 있을 것입니다! 대회에 참가하는 것은 무료입니다.'
```
Comment (marocchino) [inline ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
'Matz'는 코드를 주의깊게 테스트하고 읽어볼 것이므로, 지원할만한 충분한 가치가 있을 것입니다! 대회참가는 무료입니다.'
```

File: ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+layout: news_post
+title: "2018 후쿠오카 루비 경진대회 - Matz가 심사합니다"
```
Comment (yous) [inline ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
title: "2019 후쿠오카 루비 경진대회 - Matz가 심사합니다"
```

File: ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+translator: "shia"
+date: 2018-11-29 00:00:00 +0000
+lang: ko
+---
+
+루비스트 여러분께,
+
+일본 후쿠오카 현은 'Matz' 마츠모토 씨와 함께 여러분을 다음 경진 대회에 초대합니다. 흥미로운 루비 프로그램을 개발하셨다면 한번 지원해보세요.
+
+2018 후쿠오카 루비 경진대회 - 대상 - 백만 엔!
```
Comment (yous) [inline ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
2019 후쿠오카 루비 경진대회 - 대상 - 백만 엔!
```

File: ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md (hunk 2 above)
Comment (yous) [reply ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]: I think surrounding this line with `"` would be more appropriate than `'`. How do you think?

File: ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md (hunk 3 above)
Comment (riseshia) [reply ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]: 🙈

File: ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]: I followed this one, https://github.com/ruby/www.ruby-lang.org/pull/1872#discussion_r231775969
Is there any reason you suggest to use `"` here?

**Pull request merged**